lomap/dbmol.py

```diff
@@ -125,21 +126,27 @@ def __init__(self, directory, parallel=1, verbose='off',
             if not isinstance(display,  bool):
                 raise TypeError('The display flag is not a bool type')
 
+            if not isinstance(radial,  bool):
+                raise TypeError('The display flag is not a bool type')
             output_str=''
             display_str=''
+            radial_str=''
```

Comment:
Set these to `None` initially? e.g. `radial_str = None`? Just asking. (I realize the first two of these are not your code.)